src/plugins/data/server/server.api.md

```diff
@@ -33,6 +33,7 @@ import { ISearchOptions as ISearchOptions_2 } from 'src/plugins/data/public';
 import { ISearchSource } from 'src/plugins/data/public';
 import { IUiSettingsClient } from 'src/core/server';
 import { KibanaRequest } from 'src/core/server';
+import { KibanaRequest as KibanaRequest_2 } from 'kibana/server';
```

Comment:
I guess importing from `kibana/server` vs `src/core/server` will create a new/duplicate import in the generated docs. From what I can see this isn't a problem, but TIL.